packages/shared/lib/profile.ts

```diff
@@ -104,7 +104,7 @@ export const activeProfile: Readable<Profile | undefined> = derived(
 )
 
 activeProfileId.subscribe((profileId) => {
-    Electron.updateActiveProfile(profileId)
+    Electron?.updateActiveProfile(profileId)
```

Comment:
This change exists because it was throwing errors of reading `updateActiveProfile` of `undefined`.